packages/pharos-site/src/pages/index.module.css

```diff
@@ -185,3 +159,75 @@ figure {
   margin-bottom: var(--pharos-spacing-three-quarters-x);
   column-gap: var(--pharos-spacing-three-quarters-x);
 }
+
+@media screen and (min-width: 360px) and (max-width: 767px) {
+  .hero {
+    margin: 0 -1.25rem;
+  }
+}
+
+@media screen and (min-width: 768px) and (max-width: 1055px) {
+  .hero {
+    margin: 0 -1.5rem;
+  }
+}
+
+@media screen and (min-width: 1056px) and (max-width: 1359px) {
+  .hero {
+    margin: 0 -3.75rem;
+  }
+}
```

Comment:
Does our SCSS implementation allow for nested media queries? Could put the `.hero` rule on the outside of all the margin definitions if so.

Reply:
Yeah that would have worked. Refactored to use grid areas and the layout's `top` slot so that these custom margins won't be needed.